src/pybind/mgr/dashboard/frontend/tslint.json

```diff
@@ -112,7 +113,12 @@
           }
         ]
       }
-    ]
+    ],
+    "template-accessibility-alt-text": true,
+    "template-accessibility-tabindex-no-positive": true,
+    "template-accessibility-table-scope": true,
+    "template-accessibility-valid-aria": true,
+    "template-no-distracting-elements": true
```

Comment:
Nice! Are you aware of any well-know `tslint.conf` (like [Microsoft's tslint.conf](https://github.com/microsoft/tslint-microsoft-contrib)) where these are enabled by default?

Reply:
Unfortunately no, these rules are opt in only. One repo where they are mostly mentioned is [ tslint-to-eslint-config](https://github.com/typescript-eslint/tslint-to-eslint-config)